crates/sui-protocol-config/src/lib.rs

```diff
@@ -454,6 +524,17 @@ impl ProtocolConfig {
         // To change the values here you must create a new protocol version with the new values!
         match version.0 {
             1 => Self {
+                max_tx_in_batch: Some(10),
+                max_modules_in_publish: Some(8),
+                max_arguments: Some(16),
+                max_type_arguments: Some(128),
```

Comment:
64?

Reply:
I might go even lower here since I've never seen useful code with more than single digit type arguments (16?)

Reply:
Always happy to go lower, done :)